cheroot/server.py

```diff
@@ -972,10 +972,8 @@ def respond(self):
             self.rfile = KnownLengthRFile(self.conn.rfile, cl)
 
         self.server.gateway(self).respond()
+        self.ensure_headers_sent()
```

Comment:
It looks like you've lost ready check here. Could be:
```python
self.ready and self.ensure_headers_sent()
```

Is this intentional? What is the effect of a such change?

Reply:
Not intentional, missed that, thanks for spotting it!